src/main/java/buchen/fallingsand/Sand.java

```diff
@@ -0,0 +1,54 @@
+package buchen.fallingsand;
+
+public class Sand {
+    private final int[][] field = new int[3][3];
+
+    public String toString()
+    {
+        StringBuilder builder = new StringBuilder();
+
+        for (int y = 0; y < 3; y++) {
+            for (int x = 0; x < 3; x++) {
+                builder.append(field[y][x]);
+            }
+            builder.append("\n");
+        }
+        return builder.toString();
+    }
+
+    /**
+     * @return the value in field
+     */
+    public int get(int x, int y)
+    {
+        return field[y][x];
+    }
+
+    /**
+     * Sets the value in field to be 1
+     */
+    public void put(int x, int y)
+    {
+        field[y][x] = 1;
+    }
+
+    public void fall() {
+        //moves all sand down one square
+        //if sand at bottom level doesn't go down
+        //if sand falls and sand below it cant go down
+        boolean fell = false;
+        for (int y = 0; y < 3; y++) {
+            for (int x = 0; x < 3; x++) {
+                if (field [y][x] == 1 && y < 2 && field [y + 1][x] == 0) {
+                    put(x, y + 1);
+                    field [y][x] = 0;
+                    fell = true;
```

Comment:
Why?

Reply:
The for loop goes through the whole array until the sand can't move down anymore (It brings the sand down bit by bit until it reaches the last box it can be in) this fails the test because in the test its only meant to go down one unit, not continue to go down until it reaches a point where it can't. I resolved this by making a an if statement that once it is removed it breaks out of the loop

Reply:
All sand needs to move down when you call fall().
You can't prematurely break out of a loop.